### PR TITLE
WLT record calculation fixed

### LINKED ISSUE #1549: Differentiate Overall Record From Official Events
As suggested: http://www.chiefdelphi.com/forums/showthread.php?p=1591740#post1591740

## Affected files:
 - [Records are calculated here](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/renderers/team_renderer.py#L55). We will want to only run the overall record calculation for official events

### PR DESCRIPTION
**Pull request by maherb**
Closes #1549. Season WLT records no longer include offseasons, including those with the official flag set to true.

### CODE REVIEW

File: renderers/team_renderer.py [hunk 1]
```diff
@@ -61,7 +63,9 @@ def render_team_details(cls, handler, team, year, is_canonical):
                 qual_avg = None
                 elim_avg = None
                 wlt = EventHelper.calculateTeamWLTFromMatches(team.key_name, event_matches)
-                year_wlt_list.append(wlt)
+                print event.event_type_enum
```
Comment (phil-lopreiato) [inline renderers/team_renderer.py]: You should probably remove this

Comment (maherb) [inline renderers/team_renderer.py]: Oops, leftover from figuring out what property to use. Removed.

**Comment by phil-lopreiato** [comment]
LGTM. Can you add a screenshot?

**Comment by maherb** [comment]
image

**Comment by phil-lopreiato** [comment]
@fangeugene how do you feel about the wording - do you think it should say "Team 123 was 2-1-1 in 2016 official events" or is this fine?

**Comment by maherb** [comment]
Maybe something like "Team 123 was 2-1-1 during the 2016 official season"?

**Comment by phil-lopreiato** [comment]
That works for me. I think it's best to be extra clear about what the numbers mean. If you're feeling ambitions, maybe we could put: "Team 123 was 2-1-1 during the 2016 official season (and 3-2-1 overall)"

**Comment by maherb** [comment]
image

**Comment by maherb** [comment]
Maybe "Team #123 was 2-1-1 during official play in 2016 (and 3-2-2 overall)."?

**Comment by phil-lopreiato** [comment]
Love it!

**Comment by phil-lopreiato** [comment]
Now that I think about this, there are a couple more edge cases to handle:
 - A team has only played unofficial events (at the point in time after a team play in a week 0 scrimmage and before they play their first official event)
 - A team has only played official events (no offseason events).

Can you account for those scenarios (you can use [if statements](http://jinja.pocoo.org/docs/dev/templates/#if) in templates, also) and add some screenshots?

**Comment by maherb** [comment]
The second case has been accounted for already. I will work on the first one and post screenshots later.

**Comment by maherb** [comment]
Team with season + offseason events:
image
Season only:
image
Offseason only:
image

**Comment by maherb** [comment]
Season + offseason:
image

Just season:
image

Just offseason:
image

**Comment by maherb** [comment]
Which of these sounds best?
"Team #123 was 2-1-1 during 2016 official play and 3-2-1 overall"
"Team #123 was 2-1-1 during official play in 2016 and 3-2-1 overall"
"Team #123 was 2-1-1 during official play and 3-2-1 overall in 2016"
"In 2016, Team #123 was 2-1-1 during official play and 3-2-1 overall"
Or some combination thereof?

**Comment by fangeugene** [comment]
I like "Team #123 was 2-1-1 in official play and 3-2-1 overall in 2016"
The least important part (2016) is last.

**Comment by phil-lopreiato** [comment]
> I like "Team #123 was 2-1-1 in official play and 3-2-1 overall in 2016"
The least important part (2016) is last.

Ditto

**Comment by maherb** [comment]
Done

image

image

image

**Comment by phil-lopreiato** [comment]
👍

**Pull request merged**